recipes/newrelic/elasticsearch/rhel.yml

```diff
@@ -0,0 +1,131 @@
+  - elasticsearch
+
+# Matches partial list of the Log forwarding parameters
+logMatch:
+  - name: elasticsearch
+    file: /var/log/elasticsearch/elasticsearch*.log 
+
+validationNrql: "SELECT count(*) from ElasticsearchClusterSample where hostname like '{{.HOSTNAME}}%' FACET entityGuid SINCE 10 minutes ago"
+
+inputVars:
+  - name: "NR_CLI_API_USERNAME"
+    prompt: "Please enter your ElasticSearch username used to connecto to API."
+  - name: "NR_CLI_API_PASSWORD"
+    prompt: "Please enter your ElasticSearch username used to connecto to API."
+  - name: "NR_CLI_API_HOSTNAME"
+    prompt: "Please enter your ElasticSearch hostname below. If none is provided then the default value: localhost, will be used."
+    default: "localhost"
+  - name: "NR_CLI_API_PORT"
+    prompt: "Please enter your ElasticSearch port below. If none is provided, the default value: 9200, will be used."
+    default: 9200
+  - name: "NR_CLI_CONFIG_PATH"
+    prompt: "Please provide the path to ElasticSearch config file, if none is provided then the default: /etc/elasticsearch/elasticsearch.yml will be used."
+    default: "/etc/elasticsearch/elasticsearch.yml"
+  - name: "NR_CLI_API_USE_SSL"
+    prompt: "Do you want to use SSL to connect?"
+    default: false
+  - name: "NR_CLI_API_CA_BUNDLE_DIR"
+    prompt: "Please enter the directory in which the SSL certificate is located on the host. If using SSL please leave this field empty."
+  - name: "NR_CLI_API_CA_BUNDLE_FILE"
+    prompt: "Please enter the path to the SSL certificate file on the host. If using SSL please leave this field empty."
+
+install:
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: setup
+
+    setup:
+      label: "Installing elasticsearch integration..."
+      cmds:
+        - |
+          sudo mkdir -p "/etc/newrelic-infra/integrations.d"
+        - |
+          sudo yum update -y
+        - |
+          sudo yum install nri-elasticsearch -y
+        - |
+          if [ -f /etc/newrelic-infra/integrations.d/elasticsearch-config.yml ]; then
+            sudo rm /etc/newrelic-infra/integrations.d/elasticsearch-config.yml;
+          fi
+
+          sudo touch /etc/newrelic-infra/integrations.d/elasticsearch-config.yml;
+        - |
+          if [ "$NR_CLI_API_USE_SSL" == false ]; then
+            sudo tee -a /etc/newrelic-infra/integrations.d/elasticsearch-config.yml > /dev/null <<"EOT"
+          integration_name: com.newrelic.elasticsearch
+
+          instances:
+            - name: elasticsearch
+              command: all
+              arguments:
+                cluster_environment: staging
+                config_path: {{.NR_CLI_API_CONFIG_PATH}}
+                hostname: {{.NR_CLI_API_HOSTNAME}}
+                local_hostname: localhost
+                username: {{.NR_CLI_API_USERNAME}}
+                password: {{.NR_CLI_API_PASSWORD}}
+                port: {{.NR_CLI_API_PORT}}
+                timeout: 30
+                use_ssl: false
```

Comment:
@JakubKotkowiak I believe we should be using the input param NR_CLI_API_USE_SSL here.